lib/providers/node.js

```diff
@@ -20,18 +20,26 @@ const checkVersionFiles = filePath => {
   return filesPath
 }
 
-module.exports = function (filePath) {
+module.exports = function(filePath) {
   this.filePath = checkVersionFiles(filePath)
 
   this.readVersion = () => {
     const pkg = JSON.parse(cat(this.filePath[0]))
+    if (!pkg.version)
+      throw new Error(
+        `Your package.json file is missing the version property. If you don't need it, you can add 'version: 0.0.0' and we gonna use the version on the manifest instead!`
```

Comment:
"We are going to use" - and I'd rather suggest "null" which is more semantic than "0.0.0" :)